### ISSUE TITLE
Start leveraging new React components to replace existing global constants or other dynamic values

### ISSUE THREAD

**Issue by alfarok** [w3f/polkadot-wiki#3536]
This issue builds off of #3533 which was addressed in #3535.

The first step was to propose a implementation and this task involves leveraging the previous work to start replacing the existing constants or add new constants that have greater short-term ranges with these new components.

@DrW3RK suggested to start with these values:
```
Min Active Nomination (very useful for users landing on our Staking page)
Min stake backing an active validator in an era (resourceful for new node operators eyeing our ecosystem)
Number of parachains on Polkadot/Kusama
number of active nominators and Total number of nominators
number of active validators/ waiting validators
era number
Staked DOT/KSM percentage
```

**Comment by alfarok** [comment]
I started working on this by initially trying to swap out as many global constants as possible for components.  The work-in-progress branch can be found [here](https://github.com/w3f/polkadot-wiki/compare/%233536?expand=1).

**Comment by alfarok** [comment]
Addressed with #3557
